### ISSUE TITLE
some suggestions about copy functionality

### ISSUE THREAD

**Issue by zaqai** [binwiederhier/ntfy#715]
Android app: Add auto copy function. If ntfy received a message, it can copy the msg automatically. It will be more friendly when sharing text between devices. So far I can achieve it through macrodroid. But it is more reasonable to achieve it in ntfy.

web app: Add a copy button in the end of each message item.

**Comment by binwiederhier** [comment]
I like this idea. It would need a toggle in the settings though and would be off by default

`[ ] Copy incoming messages to clipboard automatically`

**Comment by wunter8** [comment]
Are you imagining this as a global setting or a "per-subscription" setting? It seems like it would be nice to make it per-subscription so that (for example) I don't get a copy of the ntfy stats on my clipboard twice a day

**Comment by binwiederhier** [comment]
Yes, a per-subscription only setting. Of course. A global setting would not make sense IMHO.

**Comment by wunter8** [comment]
Another option would be a per-message `X-Clipboard: true` header, but I guess that would just be ignored on web, so it's probably best to keep it just in the Android client
